aws/logs_monitoring/trace_forwarder/internal/apm/testdata/basic.json~snapshot

```diff
@@ -87,12 +89,13 @@
     Start: (int64) 1586269922931758000,
     Duration: (int64) 254812000,
     Error: (int32) 0,
-    Meta: (map[string]string) (len=5) {
+    Meta: (map[string]string) (len=6) {
      (string) (len=10) "_dd.origin": (string) (len=6) "lambda",
      (string) (len=10) "cold_start": (string) (len=5) "false",
      (string) (len=12) "function_arn": (string) (len=68) "arn:aws:lambda:us-east-1:172597598159:function:hello-dog-dev-hello36",
      (string) (len=10) "request_id": (string) (len=36) "148e0567-902c-46e3-9374-d9642152420a",
-     (string) (len=14) "resource_names": (string) (len=21) "hello-dog-dev-hello36"
+     (string) (len=14) "resource_names": (string) (len=21) "hello-dog-dev-hello36",
+     (string) (len=7) "service": (string) (len=10) "aws.lambda"
```

Comment:
Can we add a test case that actually overrides the service with the tag value?

Reply:
Good call